### DISCUSSION TITLE
How do I hide the menu

### DISCUSSION THREAD

**Discussion by tuzbtla** [labwc/labwc#2349] [Q&A] [answered]
How do I get this menu to not show up when I click on the blank, because I'm using the touchscreen on the Raspberry PI, and then it just shows an app, and I don't want the user to be able to log out. Menu in the image below

![image](https://github.com/user-attachments/assets/428f08bb-3270-471b-b0e4-23b578af7f23)

**Comment by tokyo4j** [comment, marked as answer]
Add following configuration in `<mouse>` in `~/.config/labwc/rc.xml`:
```xml
    <context name="Root">
      <mousebind button="Left" action="Press" />
      <mousebind button="Right" action="Press" />
      <mousebind button="Middle" action="Press" />
    </context>
```

Perhaps you also need to edit `/etc/xdg/labwc/rc.xml` as rpi uses `--merge-config` option (I guess you don't, as `~/.config/labwc/rc.xml` should overwrite it)

**Reply by tuzbtla** [reply]
Thank you!
